src/utils/vacations-utils.ts

```diff
@@ -107,14 +115,15 @@ export const validateUserVacationRequest = (
   vacationRequestData: { days?: number | string | null },
   currentYear: string,
   setError: (msg: string) => void,
-  setLoading: (loading: boolean) => void
+  setLoading: (loading: boolean) => void,
+  options?: {isAdmin?: boolean}
```

Comment:
This seems a little strange to add this as options and then an object containing isAdmin, why not just pass the isAdminBoolean- would be clearer.